src/libs/actions/Report.ts

```diff
@@ -3055,7 +3067,9 @@ function completeOnboarding(
     },
     adminsChatReportID?: string,
 ) {
-    const targetEmail = CONST.EMAIL.CONCIERGE;
+    const isAccountIDOdd = AccountUtils.isAccountIDOddNumber(currentUserAccountID ?? 0);
+    const targetEmail = isAccountIDOdd ? CONST.EMAIL.NOTIFICATIONS : CONST.EMAIL.CONCIERGE;
+
     const actorAccountID = PersonalDetailsUtils.getAccountIDsByLogins([targetEmail])[0];
```

Comment:
Hi, I'm working on the backend changes for this PR, and we'll need to pass the `actorAccountID` in the API request, can we update the PR for that please? Should be as simple as adding a `actorAccountID: number;` in `CompleteGuidedSetupParams.ts`, and then pass this variable below.